package NKikimrColumnShardProto;

message TBlobRange {
    optional string BlobId = 1;
    optional uint64 Offset = 2;
    optional uint64 Size = 3;
}

message TBlobRangeLink16 {
    optional uint32 BlobIdx = 1;
    optional uint64 Offset = 2;
    optional uint64 Size = 3;
}
